package.json

```diff
@@ -167,7 +167,7 @@
     "react-input-range": "^1.3.0",
     "react-markdown": "^3.1.4",
     "react-redux": "^5.0.7",
-    "react-router-dom": "4.2.2",
+    "react-router-dom": "^4.3.1",
```

Comment:
We should make sure this doesn't have any breaking changes.

Reply:
It’s a minor update so it can’t be breaking. 

lol - yeah the other package.json had the caret so the newer version was already installed there when we were getting that TS error yesterday, I think. I’ll take a look at the change log.